src/apdu_pubkey.c

```diff
@@ -69,6 +74,23 @@ void render_pkh(char *const out, size_t const out_size,
 }
 
 __attribute__((noreturn)) static void prompt_path(ui_callback_t ok_cb, ui_callback_t cxl_cb) {
+  static size_t const TYPE_INDEX = 0;
+  static size_t const DRV_PATH_INDEX = 1;
+  static size_t const ADDRESS_INDEX = 2;
+
+  static const char *const pubkey_labels[] = {
+                                              PROMPT("Provide"),
+                                              PROMPT("Derivation Path"),
```

Comment:
This isn't the prompt from the document.

Reply:
I have added #80 for the prompt